CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix 'org.apache.hc.core5.http.ParseException: Invalid protocol version' under JDK 16+ ([#4827](https://github.com/opensearch-project/OpenSearch/pull/4827))
 - Fixed compression support for h2c protocol ([#4944](https://github.com/opensearch-project/OpenSearch/pull/4944))
 - Reject bulk requests with invalid actions ([#5299](https://github.com/opensearch-project/OpenSearch/issues/5299))
+- after upgrade to OpenSearch 2.0, finished tasks no longer get stored in .tasks index ([#5376](https://github.com/opensearch-project/OpenSearch/issues/5376))
```

Comment:
Let's make this look like the other lines, please? 

"Fixed storing finished tasks in .tasks index"